examples/machine_learning/Classification_Bitter.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python
```

Comment:
`Classification_of_Bitterness_XGBoost.ipynb` is a better name

Reply:
Renamed